server/task-distributed/src/main/scala/org/apache/james/task/eventsourcing/distributed/TaskEventDTO.scala

```diff
@@ -47,14 +47,15 @@ object CreatedDTO {
 
 case class StartedDTO(@JsonProperty("type") typeName: String,
                       @JsonProperty("aggregate") aggregateId: String,
-                      @JsonProperty("event") eventId: Int)
+                      @JsonProperty("event") eventId: Int,
+                      @JsonProperty("hostname") getHostname: String)
```

Comment:
why getHostname?

Reply:
to make it (de)serializable

Reply:
that's required by jackson to make it works

Reply:
ok